superset/assets/javascripts/explorev2/components/SelectField.jsx

```diff
@@ -21,25 +26,44 @@ const defaultProps = {
 
 export default class SelectField extends React.Component {
   onChange(opt) {
-    this.props.onChange(this.props.name, opt.target.value);
+    if (this.props.multi) {
+      this.props.onChange(
+        this.props.name, opt ? opt.map((o) => (o.value)) : null);
+    } else {
+      this.props.onChange(this.props.name, opt ? opt.value : null);
+    }
   }
-
   render() {
+    const options = this.props.choices.map((c) => ({ value: c[0], label: c[1] }));
```

Comment:
the linter throws parsing error at : if {} is not wrapped with (). I think when it's () => {}, {} is recognized as a function, at the case of ()=>() it's returning object wrapped in brackets.